src/main/java/hw10/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package hw10;
+
+import java.io.File;
+import java.io.IOException;
+
+public class FileLoggerConfiguration {
+    private File file;
+    private static LoggingLevel loggingLevel;
+    private long maxFileSize;
+    private String messageFormat;
```

Comment:
Please, add a constructor that accepts all parameters and add some validation for all of them within the constructor's body.